src/cpp/core/r_util/RTokenizer.cpp

```diff
@@ -554,7 +554,7 @@ wchar_t RTokenizer::peek()
    return peek(0);
 }
 
-wchar_t RTokenizer::peek(std::size_t lookahead)
+wchar_t RTokenizer::peek(int lookahead)
```

Comment:
This change is tagging along to squelch a "comparing signed with unsigned" warning.